pkg/controller/controller.go

```diff
@@ -480,10 +480,13 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 			}
 		}
 
+		ns, _ := c.kubeClient.Core().Namespaces().Get(instance.Namespace)
```

Comment:
Careful, this assumes that the namespace already exists in kube.  Right now, you should check that there's no error and generate some new GUIDs if there is.